### PR TITLE
Add missing Materialize-CSS definitions

### PR DESCRIPTION
**Pull request by MaximBalaganskiy**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: http://materializecss.com/dropdown.html
http://materializecss.com/select.html

### CODE REVIEW

File: types/materialize-css/index.d.ts [hunk 1]
```diff
@@ -227,6 +227,128 @@ declare global {
...
+			 * The duration of the transition out in milliseconds
+			 * @default 250
+			 */
+            outDuration: number;
+
+			/**
+			 * Function called when dropdown starts entering
+			 * @default null
+			 */
+            onOpenStart: () => void;
```
Comment (huww98) [inline types/materialize-css/index.d.ts]: From Materialize source code, these callbacks are called with `this` set to the component itself and the first parameter set to the `Element`. https://github.com/Dogfalo/materialize/blob/v1-dev/js/dropdown.js#L549

Although this is not mentioned in the document, I think this is stable and can be useful sometimes

File: types/materialize-css/index.d.ts [hunk 2]
```diff
@@ -269,6 +391,52 @@ declare global {
...
+
+            /**
+             * Instance of the dropdown plugin for this select
+             */
+            dropdown: Dropdown;
+
+            /**
+             * Get selected values in an array
+             */
+            getSelectedValues(): any[];
```
Comment (huww98) [inline types/materialize-css/index.d.ts]: From my experiments, this always returns an array of string.

**Review by huww98 — CHANGES_REQUESTED** [review]
Great work! I only have some minor requests.

What I said but not mentioned in the document is optional, that is, if you does not change that part, I will approve your PR anyway.

Also, you can add your information to the header of `.d.ts` file to participate in further works.👍

**Comment by MaximBalaganskiy** [comment]
@huww98 thanks for the comments. I've made the changes. I'll post more updates later for some other components in separate PRs.

**Review by huww98 — APPROVED** [review]

**Pull request merged**